sjb/config/common/test_cases/origin_release_install_azure.yml

```diff
@@ -0,0 +1,200 @@
+---
+parent: 'common/test_cases/origin_release_with_ecosystem.yml'
+extensions:
+  sync:
+    - openshift,release=master
+  parameters:
+    - name: "FOCUS"
+      description: "Literal string to pass to <code>--ginkgo.focus</code>"
+    - name: "ADDITIONAL_SKIP"
+      description: |-
+        Regular expression to filter additional tests from the conformance suite.
+        See also:
+        <div>
+        </div>
+    - name: "OS_PUSH_BASE_REGISTRY"
+      default_value: "registry.svc.ci.openshift.org/"
+    - name: "OS_PUSH_BASE_REPO"
+      default_value: "ci-pr-images/"
```

Comment:
@smarterclayton how do we prune these today? [AFAIK `image-pruner` does prune only old tags](https://github.com/openshift/release/blob/f896f1bece92cca7b74dfbd783fec8ec63a050f5/cluster/ci/jobs/image-pruner.yaml#L33).

Reply:
that's not answered by the `oc get is -o name -n ci-pr-images | grep "${INSTANCE_PREFIX}" | xargs -r oc delete` below?

Reply:
Yes, it does, thanks